test/integration/lib/integration-server.ts

```diff
@@ -1,7 +1,7 @@
+import crypto from 'crypto'
 import Koa from 'koa'
 import bodyParser from '@koa/bodyparser'
 import http from 'http'
-import { v4 as uuid } from 'uuid'
```

Comment:
yeah I was using it it validate some id was a uuid but is was unecessary. It was used to generate a uuid but I used `crypto.randomUUID` instead. I guess normally it would be good to use the same uuid package everywhere but it was just for something throwaway in the test so i figured 1 less dependency was preferable.